libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -235,8 +231,8 @@ - (BOOL)launch
     [SFSDKCoreLogger i:[self class] format:@"Launching the Salesforce SDK."];
     _isLaunching = YES;
     self.launchActions = SFSDKLaunchActionNone;
-    if ([SFRootViewManager sharedManager].mainWindow == nil) {
-        [SFRootViewManager sharedManager].mainWindow = [SFApplicationHelper sharedApplication].windows[0];
+    if ([SFSDKWindowManager sharedManager].mainWindow == nil) {
+        [[SFSDKWindowManager sharedManager] setMainUIWindow:[SFApplicationHelper sharedApplication].windows[0]];
```

Comment:
@bhariharan mainWindow is a wrapper (SFSDKContainer). setMainUIWindow is a convenience to set the UIWindow directly. They are not the same property.